code/modules/modular_computers/file_system/programs/command/teleporter.dm

```diff
@@ -0,0 +1,118 @@
+/datum/computer_file/program/teleporter
+	filename = "teleporter"
+	filedesc = "Teleporter Control"
+	extended_desc = "A NanoTrasen command remote teleportation hub controller."
+	program_icon_state = "comm"
+	color = LIGHT_COLOR_BLUE
+	size = 8
+	requires_ntnet = TRUE
+	available_on_ntnet = FALSE
+	required_access_run = access_heads
+	usage_flags = PROGRAM_LAPTOP
+	var/obj/machinery/computer/teleporter/linked_comp
+
+/datum/computer_file/program/teleporter/ui_interact(var/mob/user)
+	var/datum/vueui/ui = SSvueui.get_open_ui(user, src)
+	if(!ui)
+		ui = new /datum/vueui/modularcomputer(user, src, "mcomputer-command-teleporter", 600, 400, "Teleporter Control System")
+		ui.auto_update_content = TRUE
+	ui.open()
+
+/datum/computer_file/program/teleporter/vueui_transfer(oldobj)
+	. = FALSE
+	var/uis = SSvueui.transfer_uis(oldobj, src, "mcomputer-command-teleporter", 600, 400, "Teleporter Control System")
+	for(var/tui in uis)
+		var/datum/vueui/ui = tui
+		ui.auto_update_content = TRUE
+		. = TRUE
```

Comment:
This enables auto update, but doesn't disable on transfer out.

Reply:
fixed